deepola/wake/src/polars_operations/merger/mapper_df.rs

```diff
@@ -39,18 +49,30 @@ unsafe impl Send for MapperDfMerger {}
 impl MapperDfMerger {
     fn new() -> Self {
         MapperDfMerger {
-            mapper: Arc::new(Box::new(|left_df, _right_df| left_df.clone())),
-            left_df: RefCell::new(None),
-            right_df: RefCell::new(None),
-            needs_left: RefCell::new(true),
-            needs_right: RefCell::new(true),
+            mapper: Arc::new(Box::new(|left_dblock, _right_df| left_dblock.clone())),
```

Comment:
Isn't `left_dblock` still a dataframe and not dblock?

Reply:
Also, minor ques: any specific reason to rename `needs_left` and `needs_right` to `need_left` and `need_right` 😅

Reply:
Just one thing I want to double-check is whether storing data-block instead of dataframe is significantly impacting query performance or not.

Reply:
Oh yes, `left_dblock`, `need_left`, and `need_right` are wrong re-renaming. I was trying different refactorings and forgot the original names. Reverted these and those in `sorted_df.rs`

About perf regression, which benchmark should I refer to? My guess is it should be faster (tho insignificant comparing to I/O) because we are not cloning left and right dataframes when storing.